js/initcontext.go

```diff
@@ -168,12 +168,12 @@ func (i *InitContext) Open(name string, args ...string) (goja.Value, error) {
 	filename := loader.Resolve(i.pwd, name)
 	data, ok := i.files[filename]
 	if !ok {
-		data_, err := loader.Load(i.fs, i.pwd, name)
+		var err error
+		data, err = afero.ReadFile(i.fs, filename)
```

Comment:
Do we have any tests for `open()` with relative file paths? Since it seems like we're not using the `pwd` parameter here, whereas it was previously used in `loader.Load()`. Quickly skimming the afero source, it doesn't seem to be concerned with the current working directory, so I think that might have to handle this ourselves?

Reply:
I don't think there is but I am probably writing some around #778 , but we do handle the pwd before the if ;)

Reply:
Ah, sorry, missed that :sweat_smile: 
But looking at `loader.Resolve()`, it seems buggy to me - it only considers files starting with `.` to be relative, which isn't the case. `open('test.txt')` should be opened as a relative path, but it currently won't be... Should I create a separate issue about that? And in any case, it seems like `open()` needs more tests...

Also, `loader` isn't probably probably the best thing to use, since we want to separate `open()` from `import`/`require` - its primary job is to be used for the latter, especially for importing URLs... The `Resolve()` method doesn't do any of that *yet*, but using it wound unnecessary couple `open()` to that loader. I think some `filepath` methods should likely be used instead.